project4/styles.css

```
@@ -0,0 +1,490 @@
+:root {
+    --color-lightgreen: #d9ef81;
+    --color-darkgreen: #b5d43c;
+    --color-offblack: #312d2f;
+    --color-offwhite: #f7f7f7;
+}
+
+/*  ================== COLOR PALETTE ==================*/
+
+*,
+*::before,
+*::after {
+    box-sizing: border-box;
+}
+
+body {
+    background-color: var(--color-offblack);
+    background-image: url("https://www.transparenttextures.com/patterns/low-contrast-linen.png");
+    color: var(--color-offwhite);
+    font-family: 'Maven Pro', sans-serif;
+    font-size: 16px;
+    line-height: 25px;
+    padding: 0 5%;
+}
+
+footer {
+    background-color: inherit;
+    background-image: inherit;
+    opacity: 60%;
```

Opacity should be represented by a decimal, like so, not a percentage:
```suggestion
    opacity: 0.6;
```